lacci/lib/shoes/app.rb

```diff
@@ -194,6 +194,10 @@ def click(&block)
     subscription_item(shoes_api_name: "click", &block)
   end
 
+  def start(&block)
+    yield self if block_given?
+  end
```

Comment:
Hm. Shoes has a start *event* that fires the first time the slot is drawn. But this seems different. It's sort of like the Slot#app method, but not exactly? Is this being used?

Reply:
hm.. ah ok sorry i misread , i just ran some example and changed code acc to the error , didn't read manual correctly my bad.

Reply:
removing this.